.github/workflows/release-please.yaml

```
@@ -0,0 +1,14 @@
+on:
+  push:
+    branches:
+      - master
+name: release-please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GoogleCloudPlatform/release-please-action@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          release-type: go-yoshi
+          package-name: go-base
```

the package name isn't actually required. This will just add it as a prefix to the release title and is more important for mono-repos 

Also, it is wrong :) 
```suggestion
          package-name: ""
```

---

Thanks for the catch :)